clearwater-cluster-manager.root/usr/share/clearwater/clearwater-cluster-manager/scripts/check_cluster_health.py

```diff
@@ -46,11 +49,17 @@
 ]
 
 local_node = sys.argv[1]
-client = etcd.Client(local_ip, 4000)
+client = etcd.Client(local_node, 4000)
 
 
 def describe_cluster(node_type, store_name):
-    result = client.get("{}/clustering/{}".format(node_type, store_name))
+    try:
+        result = client.get("{}/clustering/{}".format(node_type, store_name))
+    except etcd.EtcdKeyNotFound:
+        # There's none of the particular node type in the deployment
+        print "{} {} cluster key doesn't exist".format(node_type, store_name)
```

Comment:
I think we should probably gloss over this without printing anything (that's what the two lines below are trying to do) - I think most deployments will have quite a few node types missing (Ralf, Memento and/or Homer) and this just generates worrying but benign messages in those cases.